docs/devguide/event-conventions.asciidoc

```diff
@@ -13,6 +13,7 @@ Use the following naming conventions for field names:
 - Group related fields into subdocuments by using dot (.) notation. Groups typically have common prefixes. For example, if you have fields called `CPULoad` and `CPUSystem` in a service, you would convert
 them into `cpu.load` and `cpu.system` in the event. 
 - Avoid repeating the namespace in field names. If a word or abbreviation appears in the namespace, it's not needed in the field name. For example, instead of `cpu.cpu_load`, use `cpu.load`.
+- Use <<units,units suffix>> when the metric matches one of the known units.
```

Comment:
Should we be more explicit here on what we mean by unit suffix? Something like adding an example with `.bytes` otherwise I worry it's not clear of . or _ should be used.

Reply:
see cd2f25c32